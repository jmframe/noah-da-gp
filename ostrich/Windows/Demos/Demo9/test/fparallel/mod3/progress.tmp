          10  3.038068996663525E-018  1.634128331460261E-004
