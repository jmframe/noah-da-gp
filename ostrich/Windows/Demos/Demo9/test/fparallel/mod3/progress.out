 solve
     1      0.232E+04      0.165E-03      0.000E+00
     2      0.190E-05      0.164E-03      0.000E+00
     3      0.714E-07      0.163E-03      0.000E+00
     4      0.209E-08      0.169E-03      0.000E+00
     5      0.714E-10      0.164E-03      0.000E+00
     6      0.246E-11      0.162E-03      0.000E+00
     7      0.846E-13      0.162E-03      0.000E+00
     8      0.291E-14      0.162E-03      0.000E+00
     9      0.985E-16      0.165E-03      0.000E+00
    10      0.304E-17      0.163E-03      0.000E+00
 done
 finished
